docker/mms_docker_cpu/mxnet-model-server.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+MMS_DIR='/mxnet-model-server'
+CONFIG_FILE=$MMS_DIR'/mms_app.conf'
+GUNICORN_ARGS='gunicorn'
+MXNET_ARGS='mxnet'
+NGINX_ARGS='nginx'
+MMS_ARG='mms'
+NGINX_CONFIG_FILE='/etc/nginx/conf.d/virtual.conf'
+
+# Start MxNet model server. This function expects one argument which is the application config file.
+start_mms() 
+{
+    gunicorn_arguments=''
+    # TODO: Check if this process is already running . If yes ignore start and throw warning.
+
+    # This function expects one argument.
+    if [[ "$#" != 1 ]]
+    then
+        echo "Wrong arguments given ($#) \"$@\""
+        exit 100
+    fi
+
+    MMS_CONFIG_FILE=$1
+
+    # if the MMS CONFIG FILE is an empty string or if the file doesn't exist, throw error and exit
+    if [[ ( -z "${MMS_CONFIG_FILE##*( )}" ) || ( ! -f "$MMS_CONFIG_FILE" ) ]]
+    then
+        echo "No configuration file given... $MMS_CONFIG_FILE"
+        exit 100
+    fi
+
+    while read -r line || [ -n "$line" ]
+    do
+        line="${line##*( )}"
+	    shopt -s nocasematch
+	    # Comments and empty lines should be ignored . # and $ are comment starters.
+	    if [[ ( -z "$line" ) || ( "$line" =~ ^\# ) || ( "$line" =~ ^\$ ) ]]
+     	then
+    	    continue
+	    fi
+	
+	    if [[ "$line" =~ ^\[[A-Za-z\ ] ]] # new header
+	    then
+    	    VAR=$line
```

Comment:
Since we are revisiting, I think adding code comments will be helpful.

Reply:
Sure. I can add them.